packages/core/src/storage/migration/__tests__/__snapshots__/0.2.test.ts.snap

```diff
@@ -238,6 +238,334 @@ Object {
+      "outOfBandId": "4312fb2d-7c6f-4f02-bf0a-ef060a098fac",
+      "protocol": "https://didcomm.org/didexchange/1.0",
+      "role": "responder",
+      "state": "response-sent",
+      "theirDid": "did:peer:1zQmR2nb6CrdbGxbG2c36AN49otRFHqLnMrubav7qusHvx5W",
+      "theirLabel": "Test Agent two",
+      "threadId": "6c8714bd-da73-4772-a812-b336bcc0ae48",
+    },
+  },
+  "STORAGE_VERSION_RECORD_ID": Object {
+    "id": "STORAGE_VERSION_RECORD_ID",
+    "tags": Object {},
+    "type": "StorageVersionRecord",
+    "value": Object {
+      "createdAt": "2022-12-26T21:13:38.092Z",
+      "id": "STORAGE_VERSION_RECORD_ID",
+      "metadata": Object {},
+      "storageVersion": "0.3",
+    },
+  },
+  "cfc308b1-b515-4f4f-8f57-d86ab5b06563": Object {
+    "id": "cfc308b1-b515-4f4f-8f57-d86ab5b06563",
+    "tags": Object {
+      "connectionType": undefined,
+      "connectionTypes": Array [
+        "test-connection-type-1",
+        "test-connection-type-2",
+      ],
+      "did": "did:peer:1zQmNzAR1HL1tjy7jZgnUMmSw7i8BvLP2E6rbMXjTPWx9mfk",
+      "invitationDid": undefined,
+      "mediatorId": undefined,
+      "outOfBandId": "ab0b992a-f927-4186-9bcf-de3da1400864",
+      "role": "responder",
+      "state": "response-sent",
+      "theirDid": "did:peer:1zQmVuzEWaVi8TJLZJCeU8kBgF6gPTU9Dx8uW73Bc74CEzKw",
+      "threadId": "a1adf0d1-8ea5-43d0-ad2b-9d12fc58de9f",
+    },
+    "type": "ConnectionRecord",
+    "value": Object {
+      "autoAcceptConnection": true,
+      "connectionTypes": Array [
+        "test-connection-type-1",
+        "test-connection-type-2",
+      ],
+      "createdAt": "2022-12-26T20:51:28.442Z",
+      "did": "did:peer:1zQmNzAR1HL1tjy7jZgnUMmSw7i8BvLP2E6rbMXjTPWx9mfk",
+      "id": "cfc308b1-b515-4f4f-8f57-d86ab5b06563",
+      "metadata": Object {},
+      "outOfBandId": "ab0b992a-f927-4186-9bcf-de3da1400864",
+      "protocol": "https://didcomm.org/didexchange/1.0",
+      "role": "responder",
+      "state": "response-sent",
+      "theirDid": "did:peer:1zQmVuzEWaVi8TJLZJCeU8kBgF6gPTU9Dx8uW73Bc74CEzKw",
+      "theirLabel": "Test Agent two",
+      "threadId": "a1adf0d1-8ea5-43d0-ad2b-9d12fc58de9f",
+    },
+  },
+  "ef00d9aa-1593-4387-ad8d-0653a4f77406": Object {
+    "id": "ef00d9aa-1593-4387-ad8d-0653a4f77406",
+    "tags": Object {
+      "connectionType": undefined,
+      "connectionTypes": Array [
+        "test-connection-type-1",
+        "test-connection-type-2",
+      ],
+      "did": "did:peer:1zQmRj33HznYF4S3cj7x7C3FKeZ62njEyXforSALUcVHP6vr",
+      "invitationDid": undefined,
+      "mediatorId": undefined,
+      "outOfBandId": "b4dc56c1-71ce-46a4-8e8d-ceb8cf81b132",
+      "role": "responder",
+      "state": "response-sent",
+      "theirDid": "did:peer:1zQmQrdQkD5GZZC6JD97Y5ojZMvwNYpTe95FMXF1RHePnrgV",
+      "threadId": "7bbbb76a-cd10-432e-9b61-a322b28dc289",
+    },
+    "type": "ConnectionRecord",
+    "value": Object {
+      "autoAcceptConnection": true,
+      "connectionTypes": Array [
```

Comment:
I don't see the mediator connection type added anywhere. The test only has four connections. Maybe we also need a mediator record to see it actually adds them? Also one where we add the mediator connection type in the json beforehand and we see it doesn't add it again?

Reply:
Would this count as a valid mediator connection?

```json
{
  "id": "e171d9d6-54e1-4b28-914f-48d68c668c8b",
  "type": "ConnectionRecord",
  "value": {
    "metadata": {},
    "id": "e171d9d6-54e1-4b28-914f-48d68c668c8b",
    "createdAt": "2023-02-09T20:46:59.943Z",
    "did": "did:peer:1zQmcNtVS8NV6UFRK84ddiGPKhoyyTaZYckh1xZxMF2LAMjb",
    "invitationDid": "did:peer:2.SeyJzIjoiaHR0cHM6Ly9tZWRpYXRvci5kZXYuYW5pbW8uaWQiLCJ0IjoiZGlkLWNvbW11bmljYXRpb24iLCJwcmlvcml0eSI6MCwicmVjaXBpZW50S2V5cyI6WyJkaWQ6a2V5Ono2TWtvSG9RTUphdU5VUE5OV1pQcEw3RGs1SzNtQ0NDMlBpNDJGY3FwR25iampMcSN6Nk1rb0hvUU1KYXVOVVBOTldaUHBMN0RrNUszbUNDQzJQaTQyRmNxcEduYmpqTHEiXSwiciI6W119",
    "theirLabel": "Animo Mediator",
    "state": "completed",
    "role": "requester",
    "autoAcceptConnection": true,
    "threadId": "6e07e8a4-1598-4b06-aca4-1d4459e141cb",
    "protocol": "https://didcomm.org/didexchange/1.0",
    "outOfBandId": "ff0e1e45-fe95-4c15-b0ec-20ee5ce4df90",
    "theirDid": "did:peer:1zQmeDzifN23JzAat4CiPMy9SWbW43X7xKVUyML61EYvk9Gy"
  },
  "tags": {
    "did": "did:peer:1zQmcNtVS8NV6UFRK84ddiGPKhoyyTaZYckh1xZxMF2LAMjb",
    "threadId": "6e07e8a4-1598-4b06-aca4-1d4459e141cb",
    "invitationDid": "did:peer:2.SeyJzIjoiaHR0cHM6Ly9tZWRpYXRvci5kZXYuYW5pbW8uaWQiLCJ0IjoiZGlkLWNvbW11bmljYXRpb24iLCJwcmlvcml0eSI6MCwicmVjaXBpZW50S2V5cyI6WyJkaWQ6a2V5Ono2TWtvSG9RTUphdU5VUE5OV1pQcEw3RGs1SzNtQ0NDMlBpNDJGY3FwR25iampMcSN6Nk1rb0hvUU1KYXVOVVBOTldaUHBMN0RrNUszbUNDQzJQaTQyRmNxcEduYmpqTHEiXSwiciI6W119",
    "outOfBandId": "ff0e1e45-fe95-4c15-b0ec-20ee5ce4df90",
    "role": "requester",
    "theirDid": "did:peer:1zQmeDzifN23JzAat4CiPMy9SWbW43X7xKVUyML61EYvk9Gy",
    "state": "completed",
    "connectionType": ["test-connection-type-1", "test-connection-type-2"]
  }
}
```

Reply:
I would say so, yes!

Reply:
Ok I'll include it and run the tests